notebook/static/base/js/namespace.js

```diff
@@ -32,9 +32,9 @@ define(function(){
     // expose modules
     
     jprop('utils','base/js/utils')
-    
+  
     //Jupyter.load_extensions = Jupyter.utils.load_extensions;
-    // 
```

Comment:
Looks like you've got whitespace stripping turned on, and commits to files you didn't actually change. We would prefer to avoid committing that sort of change.